tests/testthat/test-smoke-tests.R

```diff
@@ -0,0 +1,21 @@
+library(testthat)
+library(hydromad)
+
+context("Smoke tests")
+
+data(Cotter)
+ts90s <- window(Cotter$Q, start = "1990-01-01", end = "1999-12-31")
+
+## Test that tsdiag.hydromad returns correct class
+test_that("can plot arima models with tsdiag", {
+  expect_type(tsdiag(arima(Cotter$P)), "NULL")
+  expect_type(tsdiag(arima(ts90s)), "NULL")
```

Comment:
I'm not sure if this is what we need after all - a NULL class isn't informative. Is it better just to say we expect there's no error?

Reply:
Not that tsdiag.Arima, which is called here, is defined in stats not in hydromad